### DISCUSSION TITLE
How to specify cdn of bundle.js in index.html ?

### DISCUSSION THREAD

**Discussion by sesky4** [eclipse-theia/theia#11929] [Q&A]
Currently the generated index.html is like this:
```
...
    <script type="text/javascript" src="./bundle.js" charset="utf-8"></script>
...


How can I change it to:

    <script type="text/javascript" src="https://mycdn-domain.com/bundle.js" charset="utf-8"></script>
```

I looked into source code and found that it's controlled by `FrontendGenerator.compileIndexHtml()`, which is not injectable.

Is there any way to do that ? Thanks :)

**Comment by msujew** [comment]
Hey @sesky4,

I'm not an expert on this, but can't you put the `index.html` also on a cdn? That way, both files can be loaded from cdn, especially since the `index.html` very rarely changes.
